### PR TITLE
Make it possible to change uuid via modifyLibrary

### PR DESCRIPTION
**Pull request by adriana-alter**

### CODE REVIEW

File: src/components/__tests__/TestUtils.ts [hunk 1]
```diff
@@ -66,6 +66,9 @@ export const modifyLibrary = (baseLibrary: LibraryData, newData: {[key: string]:
     let categoryToUpdate = category || null;
     let newValue = {};
     let [key, value] = pair;
+    if (Object.keys(baseLibrary).includes(key)) {
+      updatedLibrary[key] = value;
+    }
     if (!categoryToUpdate) {
       categoryToUpdate = allCategories.find((cat) => Object.keys(baseLibrary[cat]).includes(key));
     }
```
Comment (EdwinGuzman) [inline src/components/__tests__/TestUtils.ts]: This needs to be an if/else or else you'll have an extra `undefined` property:
```
{ uuid: 'UUID2',
  basic_info:
   { name: 'Test Library 1',
     short_name: 'lib1',
     description: undefined,
     number_of_patrons: '3' },
  urls_and_contact:
   { authentication_url: 'auth1',
     contact_email: 'email1',
     opds_url: 'opds1',
     web_url: 'web1' },
  stages: { library_stage: 'production', registry_stage: 'testing' },
  areas:
   { focus: [ '10025 (NY)', 'Stratford (ON)', '33154 (FL)' ],
     service: [] },
  undefined: { uuid: 'UUID2' } }
```

In the test, try console logging `updatedLibrary` after `let updatedLibrary = modifyLibrary(baseLibrary, {uuid: "UUID2"});`. This is what I originally encountered and is still happening.

Comment (EdwinGuzman) [reply src/components/__tests__/TestUtils.ts]: Actually, maybe it's not an if/else but rather change how it is added in `updatedLibrary[categoryToUpdate] = updatedCategory;`.

Comment (adriana-alter) [reply src/components/__tests__/TestUtils.ts]: It's an if/else.

**Comment by EdwinGuzman** [comment]
Great, just remove the `console.log`, fix conflicts, and use 1.3.11 so it can go bundled into the react-axe fixes. Thanks.

**Comment by adriana-alter** [comment]
I know, I realized about the console.log the minute I walked away to get coffee.  This is why I need the coffee :(

**Comment by EdwinGuzman** [comment]
:(

**Comment by adriana-alter** [comment]
I'll hold off on publishing--like you said, this can be bundled in with your fixes.

**Pull request merged**